lib/components/form/batch-styled.ts

```diff
@@ -94,13 +106,19 @@ export const BatchPreferencesContainer = styled.div`
   margin-top: 50px;
   padding: 5px 10px;
 `
-
 export const MainSettingsRow = styled.div`
   align-items: top;
   display: flex;
   flex-direction: row;
-  justify-content: flex-start;
+  justify-content: space-between;
   margin-bottom: 5px;
+
+  @media (max-width: 355px), (min-width: 991px) and (max-width: 1110px) {
```

Comment:
Can you use the bootstrap thresholds? Otherwise, add a comment to explain the choice of thresholds.